### ISSUE TITLE
CTA / Organization / Create

### ISSUE THREAD

**Issue by paulsonder** [OpenConceptLab/ocl_issues#2014]
As a user, I want to be able to **Create** and **Organization** so that I can organize content separately from my own personal namespace and collaborate with other users.

## Acceptance criteria

1. User clicks to create a new Organization
2. User completes Create Organization form

## Design docs

**Entry points**

Users are able to create an Organization by clicking on the 'Create' menu in the main **Navigation bar** or from the **Quick Links drawer**...

<img width="267" alt="Screenshot 2024-11-25 at 15 47 44" src="https://github.com/user-attachments/assets/f98be3ac-225b-47b6-ac83-08e710b9a360">

<img width="336" alt="Screenshot 2024-11-25 at 15 42 04" src="https://github.com/user-attachments/assets/836ca913-fce0-49e1-ad97-dfca94011083">

**Forms**

The Create Organization forms can be found [here](https://zpl.io/p011vMy).

**Comment by jamlung-ri** [comment]
>     1. Should we allow a user to create and manage custom attributes for an Organization? Are there use cases for this? If Yes, is it something we can push to post-launch as a topic?


>     2. Should we make uploading an avatar mandatory? If not an image, should there be a fall back with an initial or other idea to make the avatar a little more distinctive with a view to making them more visible in search results etc?

Is that common in other tools? I agree that giving users something distinctive would be good, but I don't know what the common approach is. It especially feels like it could deter users from creating a user profile if they feel like they have to input an avatar of some sort.

I could get on board with a "Hey I know you hit submit but you didn't put a profile pic! These help people know who you are, and it doesn't even have to be your picture..." approach, but allow people to bypass it if they really don't want to put something. 

Also, GitHub's might be a good example where people are given a semi-unique picture if they don't assign one themselves.

**Comment by paulsonder** [comment]
1. So maybe Custom org attributes could be displayed in the same format as FHIR attributes are in the Repo overview, using the structured list component?
2. Applications usually provide some kind of fallback, others will create an avatar from your initials (I can't imagine this option being useful for an Org as opposed to a person with first an last name). I think a logo should be encouraged, but not mandatory, with a better fallback than we're using currently.

**Comment by jamlung-ri** [comment]
> 1. So maybe Custom org attributes could be displayed in the same format as FHIR attributes are in the Repo overview, using the structured list component?
Yeah I think that could be ideal!

>     2. Applications usually provide some kind of fallback, others will create an avatar from your initials (I can't imagine this option being useful for an Org as opposed to a person with first an last name). I think a logo should be encouraged, but not mandatory, with a better fallback than we're using currently.
Agreed, encouraged but not mandatory is the ideal approach, and perhaps with an Identicon fallback? https://identicon.net/

**Comment by jamlung-ri** [comment]
Good point, I don't know if it's a good idea to have slightly different approaches between Users and Orgs, but we could stand to push a little harder to populate an avatar when creating Orgs. Even so, some sort of visual identity fallbacks for both types would be good to have nailed down.
